src/OpenIddict.EntityFramework/Models/OpenIddictApplication.cs

```diff
@@ -19,54 +20,57 @@ public class OpenIddictApplication : OpenIddictApplication<string, OpenIddictTok
     }
 
     /// <summary>
-    /// Represents an OpenIddict application.
+    /// The default implementation of <see cref="OpenIddictApplication{TKey}"/>.
     /// </summary>
     public class OpenIddictApplication<TKey> : OpenIddictApplication<TKey, OpenIddictToken<TKey>>
         where TKey : IEquatable<TKey> { }
 
     /// <summary>
-    /// Represents an OpenIddict application.
+    /// Represents an application in the OpenIddict system.
     /// </summary>
     public class OpenIddictApplication<TKey, TToken> where TKey : IEquatable<TKey> {
         /// <summary>
-        /// Gets or sets the display name
-        /// associated with the current application.
+        /// Gets or sets the primary key for this application.
+        /// </summary>
+        public virtual TKey Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets the friendly name used in a UI for this application.
         /// </summary>
         public virtual string DisplayName { get; set; }
 
         /// <summary>
-        /// Gets or sets the unique identifier
-        /// associated with the current application.
+        /// Gets or sets the unique identifier for this application.
         /// </summary>
-        public virtual TKey Id { get; set; }
+        public virtual string ApplicationId { get; set; }
```

Comment:
What is the idea behind this?
To correspond to `client_id` query parameter of the URL?